### ISSUE TITLE
Plain emacs links cause error when fontifying in backlink buffer in v2

### ISSUE THREAD

**Issue by codygman** [org-roam/org-roam#1683]
#### Steps to Reproduce

![Screenshot from 2021-07-26 14 41 18](https://user-images.githubusercontent.com/225847/127049033-bc0ac88f-4d29-494d-b16e-07bacf03699a.png)

#### Backtrace
```
  org-roam-fontify-like-in-org-mode((1 243 "[[file:.gitignore]]"))
  org-roam-node-insert-section(:source-node #s(org-roam-node :file "/home/cody/org-roam/20210726143330-investigate_pot..." :file-hash nil :file-atime (24831 3903 717109 500000) :file-mtime (24831 3897 201184 819000) :id "5b8533af-43e6-4fa8-96fd-be4c0eb74861" :level 0 :point 1 :todo nil :priority nil :scheduled nil :deadline nil :title "investigate potential bug in org-roam backlink buf..." :properties (("CATEGORY" . "20210726143330-investigate_potential_bug_in_org_ro...") ("ID" . "5b8533af-43e6-4fa8-96fd-be4c0eb74861") ("BLOCKED" . "") ("FILE" . "/home/cody/org-roam/20210726143330-investigate_pot...") ("PRIORITY" . "B")) :olp nil :tags nil :aliases nil :refs nil) :point 243 :properties (:outline nil))
  org-roam-backlinks-section(#s(org-roam-node :file "/home/cody/org-roam/20210726143524-a_new_page_to_l..." :file-hash nil :file-atime (24831 3723 698253 313000) :file-mtime (24831 3711 668338 325000) :id "60d2dd3b-91b1-452b-9076-7027731e86a7" :level 0 :point 1 :todo nil :priority nil :scheduled nil :deadline nil :title "a new page to link to" :properties (("CATEGORY" . "20210726143524-a_new_page_to_link_to") ("ID" . "60d2dd3b-91b1-452b-9076-7027731e86a7") ("BLOCKED" . "") ("FILE" . "/home/cody/org-roam/20210726143524-a_new_page_to_l...") ("PRIORITY" . "B")) :olp nil :tags nil :aliases nil :refs nil))
  run-hook-with-args(org-roam-backlinks-section #s(org-roam-node :file "/home/cody/org-roam/20210726143524-a_new_page_to_l..." :file-hash nil :file-atime (24831 3723 698253 313000) :file-mtime (24831 3711 668338 325000) :id "60d2dd3b-91b1-452b-9076-7027731e86a7" :level 0 :point 1 :todo nil :priority nil :scheduled nil :deadline nil :title "a new page to link to" :properties (("CATEGORY" . "20210726143524-a_new_page_to_link_to") ("ID" . "60d2dd3b-91b1-452b-9076-7027731e86a7") ("BLOCKED" . "") ("FILE" . "/home/cody/org-roam/20210726143524-a_new_page_to_l...") ("PRIORITY" . "B")) :olp nil :tags nil :aliases nil :refs nil))
  org-roam-buffer-render()
  funcall-interactively(org-roam-buffer-render)
  call-interactively(org-roam-buffer-render nil nil)
  command-execute(org-roam-buffer-render)

```

#### Expected Results

fontifying it doesn't fail

#### Actual Results

it fails

### Environment

- Emacs: GNU Emacs 27.2 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.27, cairo version 1.16.0)
- Framework: N/A
- Org: Org mode version 9.4.6 (9.4.6-11-g1ee52c-elpa @ /nix/store/0lr6w4yjpjzh7q22slizbpnks4d0qq8v-emacs-packages-deps/share/emacs/site-lisp/elpa/org-20210719/)
- Org-roam: 2.0.0

- ~Org-roam commit: ~ org-roam-20210721.1746

**Comment by codygman** [comment]
I wonder if #1676 might also address this.

**Comment by codygman** [comment]
This is fixed as of 6dc8dda5e59765d3e4629469f4a400b315e92a51, very likely from the linked issue above.
